pre_commit_hooks/check_case_conflict.py

```diff
@@ -12,9 +13,23 @@ def lower_set(iterable: Iterable[str]) -> Set[str]:
     return {x.lower() for x in iterable}
 
 
+def parents(file: str) -> Iterator[str]:
+    while True:
+        file = file[:file.rfind('/')]
+        if not file:
+            return
```

Comment:
this can be `while file != os.sep:` I think ?

Reply:
- `os.sep` is different on windows, git always return paths in a posix format
- paths looks like that one: `testing/resources/does_exist`. You don't have a leading `/`

Reply:
done, it seems to work